app/components/collections/edit/participant_component.html.erb

```diff
@@ -1,3 +1,3 @@
-<p class="participant-label"><%= value %></p>
+<span class="participant-label"><%= value %></span>
```

Comment:
the p tag was messing up the vertical spacing in the row, took me too long to realize this